bin/GameIndex.dbf

```diff
@@ -5180,6 +5180,12 @@ Serial = SCUS-97328
 Name   = Gran Turismo 4
 Region = NTSC-U
 Compat = 5
+//[patches = 77E61C8A]
+//
+//Fixes black screen bug that prevents game from loading
+//patch=1,EE,00100D84,word,24100001 
+//[/patches]
+//
```

Comment:
patch won't be read from the database if you use ``//`` at the beginning of the line. ( currently they're commented out)